xml/System.Collections/IComparer.xml

```diff
@@ -92,25 +92,9 @@
         <param name="x">The first object to compare.</param>
         <param name="y">The second object to compare.</param>
         <summary>Compares two objects and returns a value indicating whether one is less than, equal to, or greater than the other.</summary>
-        <returns>A signed integer that indicates the relative values of <paramref name="x" /> and <paramref name="y" />, as shown in the following table.  
+        <returns>A signed integer that indicates the relative values of <paramref name="x" /> and <paramref name="y" />. If less than 0, <paramref name="x" /> is less than <paramref name="y" />; if 0, <paramref name="x" /> equals <paramref name="y" />; if greater than 0, <paramref name="x" /> is greater than <paramref name="y" />. 
```

Comment:
this is a bit harder to parse online. did you change this because of IntelliSense?

Reply:
Yes, @mairaw. I changed it because of IntelliSense. We really need to remove tables from the return value section, since it creates a really broken IntelliSense experience (I keep meaning to open an issue -- I'll do that now). I'm not sure if this is the best format to replace tables with, but I thought that I'd get rid of the table while I was touching the file.